resources/public/playground-dbg.html

```diff
@@ -7,16 +7,18 @@
 <link rel="stylesheet" type="text/css" href="css/codemirror.css">
 </head>
 <body>
-<pre><code class="clojure">
-    "abc"
-</code></pre>
 
 <pre><code class="clojure">
-    (let [a 1]
-    a)
+    (loop [x 1]
```

Comment:
ah, i'm glad you asked that, it had slipped my mind!

the console.log call is a big perf hit, but obviously that's just for testing purposes. i removed the console.log call and ran the code from my blog post, using this playground-dbg.html: http://pastebin.com/96ix57V2 . i ran the chrome cpu profiler and pressed ctrl+enter in the bottom window ~8 times. 

if i'm reading profile output right, we do end up spending a noticeable amount of time in guard. it's difficult to give a specific number - when i try to eliminate idle time from the profile view so that i can see actually accurate data, profiler doesn't want to cooperate. best guess is that the program is 25% slower than it would have been otherwise, but i could be reading this wrong. here's what i'm seeing: https://www.evernote.com/l/AAudWEr1SsxEzZr21Ot499bPabBl4MlKd2o . the main culprit appears to be the call to (system-time).

with the console.log removed, though, this change doesn't seem to me to have a prohibitive performance impact. noticeable, but not super super bad as far as i can tell.